handler.py

```diff
@@ -147,26 +131,20 @@ def runScanFromQ(event, context):
     os.environ['PATH'] = original_pathvar
 
 
-def runScheduledObservatoryScan(event, context):
+def addScheduledObservatoryScansToQueue(event, context):
```

Comment:
Method names here are getting unweildy.  This is why I think abstracting these our into modules and object classes will be a strong dev pattern long term.

Reply:
Agree with this also, however may need your help with implementation details.

Reply:
Happy to do so,  was waiting until we were at a good place to do deal with such a big change before doing so.